torch/onnx/__init__.py

```diff
@@ -55,25 +75,28 @@ def _export(*args, **kwargs):
 
 
 def export(
-    model,
-    args,
-    f,
-    export_params=True,
-    verbose=False,
-    training=TrainingMode.EVAL,
-    input_names=None,
-    output_names=None,
-    operator_export_type=OperatorExportTypes.ONNX,
-    opset_version=None,
-    do_constant_folding=True,
-    dynamic_axes=None,
-    keep_initializers_as_inputs=None,
-    custom_opsets=None,
-    export_modules_as_functions=False,
-):
-    r"""
-    Exports a model into ONNX format. If ``model`` is not a
-    :class:`torch.jit.ScriptModule` nor a :class:`torch.jit.ScriptFunction`, this runs
+    model: Union[torch.nn.Module, torch.jit.ScriptModule, torch.jit.ScriptFunction],
+    args: Union[Tuple[Any, ...], torch.Tensor],
+    f: Union[str, BinaryIO],
```

Comment:
I'm not 100% sure with this. I tried to call `f.getvalue()` and it appears `getvalue()` is not declared for BinaryIO. Since it's marked deprecated in 3.8 and will remove in 3.12, might as well just stick to `io.BytesIO`

Reply:
Sounds good! I didn't notice it was deprecated

Reply:
Done